### DISCUSSION TITLE
Just use subpopulation as reference

### DISCUSSION THREAD

**Discussion by TerminatorJ** [BayraktarLab/cell2location#139] [Q&A] [answered]
Thank you for developing this fantastic tool!
To speed up the process of deconvolution, I guess some users want to just use part of their interesting populations as their reference. 
1)Would it make sense to use just one cell type as a reference?

2)What's the performance of subpopulation? which means if you put sub-population as a reference, the results are different from setting cell-type like T-cells and B-cells as our reference.

3)Can you interpret the biological meanings of the final results of 'q05_cell_abundance_w_sf'? if I got a number of 18 what does it means? what about 0.8? what's the difference between them? Are they comparable?

**Comment by vitkl** [comment, marked as answer]
Hi @TerminatorJ 

Thanks for using cell2location and reaching out.

Cell2location run time can be somewhat higher for a large and very large number of cell types (50-100+) - but this difference is not enough to justify using a size-reduced reference (the main factor affecting speed is data size = N genes * N cells). To get the best results with cell2location you need to cover all major cell lineages present in the spatial sample. If you have a reference that represents 1/2 regions of the tissue (e.g. adrenals from kidney cortex + adrenals) you can expect to get good cell abundance maps only for a matching region (adrenals). Also to get the best results you need to remove non-matching locations (kidney cortex) using histology info.

1. No. Cell2location needs multiple cell types and cell subpopulations (see above).

2. I take that you mean providing broad cell type labels for all cell types (e.g. 10) but one. For that one cell type, you provide subpopulation annotation (e.g. 8). We used this approach in a few projects and indeed simplifies cell annotation work. In the cell2location paper (Fig 4-5) we should that cell2location is especially good for mapping fine-grained subtypes - but you always need to critically assess the results, compare them with literature, orthogonal methods.

3. These values represent approximate cell abundance - the expected number of cells of a given cell type at each location. This number can be fractional (e.g. 0.8) because you can have a part of the cell captured in a 2D section. In general, we see good concordance with prior knowledge (e.g. in the LN tutorial), however, in some cases, it is more meaningful to only compare how cell abundance of one cell type varies within a section rather than comparing abundances of cell types to each other.
